app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,366 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+ @import "bootstrap-sprockets";
+ @import "bootstrap";
+
+#nav.affix {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index:10;
+}
+
+.navbar.scrolled {
+  background: rgb(255, 255, 255); //IE
+  background: rgba(248, 248, 248, 0.9); //NON-IE
+  -webkit-transition: all 0.6s ease;
+  -moz-transition: all 0.6s ease;
+  -o-transition: all 0.6s ease;
+  -ms-transition: all 0.6s ease;
+  transition: all 0.6s ease;
+}
+
+.nav-center {
+  margin:0;
+  float:none;
+}
+
+.navbar-inner{
+  text-align:center;
+}
+
+.navbar {
+  position: relative;
+  background: #FFFFFF;
+  -webkit-transition: all 0.6s ease;
+  -moz-transition: all 0.6s ease;
+  -o-transition: all 0.6s ease;
+  -ms-transition: all 0.6s ease;
+  transition: all 0.6s ease;
+}
+
+.navbar-right {
+  margin-right: 0px;
+}
+
+.brand {
+  position: absolute;
+  left: 50%;
+  margin-left: -275px !important;  /* 50% of your logo width */
+  margin-top: -52px;
+  display: block;
+}
+
+.jumbotron {
+   height: 200px;
+   background: #FFFFFF;
+   margin-bottom: 0px;
+}
+
+.logo {
+   position: absolute;
+   left: 0px;
+   top: 24px;
+}
+
+.main-content {
+  background: #000000;
+  background-image: image-url("galaxy.jpeg");
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.video {
+  text-align: center;
+  margin-bottom: 10px;
+  iframe {
+    border-left: 0px;
+    border-right: 0px;
+    margin-left: -25px;
+  }
+}
+
+.row-centered {
+  text-align: center;
+  margin-left: 25px;
+}
+
+.fm-row-centered {
+  text-align: center;
+}
+
+@media (min-width: 1200px){
+  .fm-row-centered {
+    margin-left: 90px;
+  }
+}
+
+.c-search {
+  background-color: rgba(255, 255, 255, 0.9);
+  border-radius: 15px;
+  margin: 15px;
+  padding: 10px;
+  text-align: center;
+  display:inline-block;
+  width: 240px;
+  overflow: scroll;
+}
+
+@media (min-width: 992px){
+  .c-search {
+    height: 145px;
+  }
+}
+
+.display-img {
+  border-radius: 50%;
+}
+
+.form-control:focus {
+  border-color: #EDA83C;
+  box-shadow: 0px 1px 1px #EDA83C inset, 0px 0px 8px #EDA83C;
+}
+
+:focus  {
+  outline: 0 !important;
+  outline-color: #EDA83C;
+  border-color: #EDA83C;
+  box-shadow: 0px 1px 1px #EDA83C inset, 0px 0px 8px #EDA83C;
+}
+
+a:focus {
```

Comment:
On styles that effect base elements(p, h1, a, etc.) it's a good idea to move them to the top of your stylesheet.